packages/babel-parser/test/fixtures/typescript/class/accessor-invalid/input.ts

```diff
@@ -0,0 +1,15 @@
+abstract class Foo {
+  declare accessor prop7: number;
+  private accessor #p: any;
+
+  accessor a!;
```

Comment:
TS will throw `Declarations with definite assignment assertions must also have type annotations.`, we can address that in a separate PR.